include/SDL3/SDL_init.h

```diff
@@ -105,7 +104,7 @@ extern "C" {
  * \sa SDL_SetMainReady
  * \sa SDL_WasInit
  */
-extern DECLSPEC int SDLCALL SDL_Init(Uint32 flags);
+extern DECLSPEC int SDLCALL SDL_Init(SDL_InitFlags flags);
```

Comment:
This should stay Uint32, as it's a bitmask of flags, and compilers tend to complain if you try to stuff OR'd enums into an enum.